workloads/sample/sample.sh

```diff
@@ -19,9 +20,9 @@ run() {
 validate() {
     export PYTHONPATH=${wt_build_dir}/lang/python:${wt_home_dir}/tools:$PYTHONPATH
     echo "Running verify..."
-    ${wt_build_dir}/wt -h "$1/$database_dir" -R verify
+    ${wt_build_dir}/wt -h "$1/$database_dir" -R verify 2>&1 | sudo tee ${failure_dir}/output.txt
```

Comment:
We might need to remove `set -e` to have this to work, but we could do this:
```
if ! ${wt_build_dir}/wt -h "$1/$database_dir" -R verify 2>&1 | sudo tee ${failure_dir}/output.txt; then sudo rm ${failure_dir}/output.txt; done
```

Reply:
This looks logically the same?